src/ui/reducers/index.ts

```diff
@@ -32,6 +33,7 @@ export const selectors = {
   ...consoleSelectors,
   ...debuggerSelectors,
   ...eventListenerBreakpointsSelectors,
+  ...hitCountsSelectors,
```

Comment:
this is nice to have because it lets me evaluate `app.selectors.getHitCountsForSource` in the console

Reply:
:speech_balloon: The one issue here is that this also ends up including the Redux action creators, which aren't actually selectors :)

Granted we've had that happen with other files too, but going forward I'd like to stick with what I did over in the new `ui/reducers/sources` file and explicitly define an object with the right selectors inside, which can then be imported and spread.

Reply:
@markerikson that works for me. Just love having an `app.selectors`